src/Aspire.Hosting.Kafka/KafkaBuilderExtensions.cs

```diff
@@ -169,5 +170,7 @@ private static void ConfigureKafkaContainer(EnvironmentCallbackContext context,
             $"PLAINTEXT://{primaryEndpoint.Property(EndpointProperty.Host)}:29092,PLAINTEXT_HOST://{primaryEndpoint.Property(EndpointProperty.Host)}:{primaryEndpoint.Property(EndpointProperty.Port)},PLAINTEXT_INTERNAL://{internalEndpoint.Property(EndpointProperty.Host)}:{internalEndpoint.Property(EndpointProperty.Port)}");
 
         context.EnvironmentVariables["KAFKA_ADVERTISED_LISTENERS"] = advertisedListeners;
+
+        context.EnvironmentVariables["KAFKA_LOG_DIRS"] = Target;
```

Comment:
We only want to do this if `WithDataVolume` or `WithDataBindMount` was called, right?

Reply:
I think so.